.travis.yml

```diff
@@ -1,16 +1,21 @@
-sudo: false
+sudo: required # due to manual install of make 4.1
```

Comment:
Looks like a full VM still if sudo is set to required.

https://docs.travis-ci.com/user/reference/overview/

They could still be abstracting things with Docker on the VM to make it easer to deploy.